integration-tests/proguard-rules.pro

```diff
@@ -19,4 +19,4 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
--keep class com.revenuecat.purchases.** { *; }
\ No newline at end of file
+-keep class com.revenuecat.purchases.** { *; }
```

Comment:
Right, I was thinking we probably wanted to remove the integration tests module soon. But it's true this now allow us to run some basic tests against a minified version of the SDK that we wouldn't have otherwise,